### PR TITLE
Update clippy.json

### PR DESCRIPTION
**Pull request by KnyajeSatalin**

### CODE REVIEW

**Comment by KnyajeSatalin** [comment]
Zup) Can you confirm changes? I want to add funny comments on russian site "vk.com'.

File: clippy.json [hunk 1]
```diff
@@ -63,5 +63,11 @@
     "mail.google.com": [
         "Do you need glasses?",
         "Maybe try Outlook? wink wink..."
+    ],
+    "vk.com": [
+        "Дуров, верни стену.",
+        "При помощи фена и простой ванны, можно сделать отличное джакуззи.."
```
Comment (capJavert) [inline clippy.json]: No worries :)

Each line of JSON file should end with `,` (comma), except for the last line. You can use https://jsonlint.com to validate contents of `clippy.json` before submitting again.

**Review by capJavert — CHANGES_REQUESTED** [review]
JSON not valid.

File: clippy.json (hunk 1 above)
Comment (KnyajeSatalin) [reply clippy.json]: Oke. :)

**Review by capJavert — APPROVED** [review]

**Comment by capJavert** [comment]
Thanks @KnyajeSatalin

**Pull request merged**